--- OpenIPMI-2.0.18/configure.ac.wiget	2011-08-15 14:56:33.536069856 +0200
+++ OpenIPMI-2.0.18/configure.ac	2011-08-15 15:05:50.412874577 +0200
@@ -647,7 +647,13 @@ else
 fi
 if test "x$pythonprog" != "x"; then
    if test "x$pythonusepthreads" = "x"; then
-      pythonusepthreads=`echo -e "try:\n  import thread\n  print 'yes'\nexcept:\n  print 'no'\n" | python`
+      pythonusepthreads=`$pythonprog <<EOF
+try:
+	import thread
+	print 'yes'
+except:
+	print 'no'
+EOF`
    fi
    echo "checking for python threads... $pythonusepthreads"
 fi
@@ -671,7 +677,13 @@ AC_SUBST(PYTHON_POSIX_SO)
 # Check for Tkinter/Tix
 if test "x$tkinter" = "x"; then
    if test "x$pythonprog" != "x"; then
-      tkinter=`echo -e "try:\n  import Tix\n  print 'yes'\nexcept:\n  print 'no'\n" | python`
+      tkinter=`$pythonprog <<EOF
+try:
+	import Tix
+	print 'yes'
+except:
+	print 'no'
+EOF`
    fi
 fi
 echo "checking for tkinter... $tkinter"
