--- OpenIPMI-1.4.12/unix/Makefile.am.orig	2005-02-02 21:50:02.000000000 +0100
+++ OpenIPMI-1.4.12/unix/Makefile.am	2005-02-27 23:25:07.014728752 +0100
@@ -16,7 +16,7 @@
 
 libOpenIPMIpthread_la_SOURCES = posix_thread_os_hnd.c selector.c
 libOpenIPMIpthread_la_LIBADD = -lpthread $(GDBM_LIB) \
-	$(top_builddir)/utils/libOpenIPMIutils.la
+	$(top_builddir)/utils/libOpenIPMIutils.la $(top_builddir)/lib/libOpenIPMI.la
 libOpenIPMIpthread_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
 	-Wl,-Map -Wl,libOpenIPMIpthread.map
 
--- OpenIPMI-2.0.0/cmdlang/Makefile.am.orig	2005-03-10 16:58:13.000000000 +0100
+++ OpenIPMI-2.0.0/cmdlang/Makefile.am	2005-04-16 02:40:01.125790968 +0200
@@ -19,6 +19,7 @@
 	cmd_pef.c cmd_conn.c cmd_fru.c out_fru.c os_debug.c
 libOpenIPMIcmdlang_la_LIBADD = -lm \
 	$(top_builddir)/utils/libOpenIPMIutils.la \
+	$(top_builddir)/unix/libOpenIPMIposix.la \
 	$(top_builddir)/lib/libOpenIPMI.la
 libOpenIPMIcmdlang_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
 	-Wl,-Map -Wl,libOpenIPMIcmdlang.map
