--- OpenIPMI-1.4.12/unix/Makefile.am.orig	2005-02-02 21:50:02.000000000 +0100
+++ OpenIPMI-1.4.12/unix/Makefile.am	2005-02-27 23:25:07.014728752 +0100
@@ -16,7 +16,7 @@
 
 libOpenIPMIpthread_la_SOURCES = posix_thread_os_hnd.c selector.c
 libOpenIPMIpthread_la_LIBADD = -lpthread $(GDBM_LIB) \
-	$(top_builddir)/utils/libOpenIPMIutils.la
+	$(top_builddir)/utils/libOpenIPMIutils.la $(top_builddir)/lib/libOpenIPMI.la
 libOpenIPMIpthread_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
 	-Wl,-Map -Wl,libOpenIPMIpthread.map
 
--- OpenIPMI-2.0.1/ui/Makefile.am.orig	2005-03-31 22:57:41.000000000 +0200
+++ OpenIPMI-2.0.1/ui/Makefile.am	2005-06-26 21:44:21.903589616 +0200
@@ -22,7 +22,7 @@
 lib_LTLIBRARIES = libOpenIPMIui.la
 
 libOpenIPMIui_la_SOURCES = ui.c ui_os.c ui_keypad.c ui_command.c 
-libOpenIPMIui_la_LIBADD =
+libOpenIPMIui_la_LIBADD = $(top_builddir)/lib/libOpenIPMI.la $(top_builddir)/unix/libOpenIPMIposix.la -lncurses
 libOpenIPMIui_la_LDFLAGS = -version-info $(LD_VERSION) \
 	-Wl,-Map -Wl,libOpenIPMIui.map
 
--- OpenIPMI-2.0.0/cmdlang/Makefile.am.orig	2005-03-10 16:58:13.000000000 +0100
+++ OpenIPMI-2.0.0/cmdlang/Makefile.am	2005-04-16 02:40:01.125790968 +0200
@@ -19,6 +19,7 @@
 	cmd_pef.c cmd_conn.c cmd_fru.c out_fru.c os_debug.c
 libOpenIPMIcmdlang_la_LIBADD = -lm \
 	$(top_builddir)/utils/libOpenIPMIutils.la \
+	$(top_builddir)/unix/libOpenIPMIposix.la \
 	$(top_builddir)/lib/libOpenIPMI.la
 libOpenIPMIcmdlang_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
 	-Wl,-Map -Wl,libOpenIPMIcmdlang.map
--- OpenIPMI-2.0.7/configure.in.orig	2006-06-08 21:50:40.000000000 +0200
+++ OpenIPMI-2.0.7/configure.in	2006-08-22 23:54:55.736467750 +0200
@@ -752,9 +752,10 @@
    *-sun-*) AC_DEFINE([_SUNOS], [], [Solaris's term.h does horrid things.]);;
 esac
 
-AC_CHECK_LIB(curses, tgetent, TERM_LIBS=-lcurses,
+AC_CHECK_LIB(tinfo, tgetent, TERM_LIBS=-ltinfo,
+[AC_CHECK_LIB(curses, tgetent, TERM_LIBS=-lcurses,
    [AC_CHECK_LIB(ncurses, tgetent, TERM_LIBS=-lncurses,
-      [AC_MSG_ERROR([libtermcap, libcurses or libncurses are required!])] )] )
+      [AC_MSG_ERROR([libtermcap, libcurses or libncurses are required!])] )] )])
 
 # Checks for header files.
 AC_FUNC_ALLOCA
