--- OpenIPMI-2.0.28/configure.ac.orig	2019-12-12 17:39:40.000000000 +0100
+++ OpenIPMI-2.0.28/configure.ac	2020-02-14 07:15:42.849858578 +0100
@@ -369,7 +369,7 @@
    FOUND_TCL_HEADER=no
    ver=`echo 'puts \$tcl_version' | tclsh`
    if test "x$tclcflags" = "x"; then
-      AC_CHECK_HEADER(tcl/tcl.h, FOUND_TCL_HEADER=yes; )
+      AC_CHECK_HEADER(tcl.h, FOUND_TCL_HEADER=yes; )
       if test "x$FOUND_TCL_HEADER" != "xyes"; then
          AC_CHECK_HEADER(tcl/tcl.h, FOUND_TCL_HEADER=yes; )
 	 if test "x$FOUND_TCL_HEADER" == "xyes"; then
--- OpenIPMI-2.0.37/tcl/tcl_os_hnd.c	2024-04-18 03:30:21.000000000 +0200
+++ OpenIPMI-2.0.37/tcl/tcl_os_hnd.c	2026-05-22 11:54:41.151366398 +0200
@@ -374,7 +374,7 @@
 {
     if (cond->waiters)
 	return EBUSY;
-    Tcl_ConditionFinalize(cond->cond);
+    Tcl_ConditionFinalize(&cond->cond);
     free(cond);
     return 0;
 }
