--- OpenIPMI-2.0.28/configure.ac.orig	2019-12-12 17:39:40.000000000 +0100
+++ OpenIPMI-2.0.28/configure.ac	2020-02-14 07:15:42.849858578 +0100
@@ -369,7 +369,7 @@
    FOUND_TCL_HEADER=no
    ver=`echo 'puts \$tcl_version' | tclsh`
    if test "x$tclcflags" = "x"; then
-      AC_CHECK_HEADER(tcl/tcl.h, FOUND_TCL_HEADER=yes; )
+      AC_CHECK_HEADER(tcl.h, FOUND_TCL_HEADER=yes; )
       if test "x$FOUND_TCL_HEADER" != "xyes"; then
          AC_CHECK_HEADER(tcl/tcl.h, FOUND_TCL_HEADER=yes; )
 	 if test "x$FOUND_TCL_HEADER" == "xyes"; then
