--- OpenIPMI-2.0.14/swig/perl/Makefile.am.orig	2007-10-02 16:34:34.000000000 +0200
+++ OpenIPMI-2.0.14/swig/perl/Makefile.am	2008-07-10 07:29:13.967027928 +0200
@@ -16,19 +16,19 @@
 all: OpenIPMI.so
 
 OpenIPMI_SRC = OpenIPMI_wrap.c OpenIPMI_perl.c
-OpenIPMI_OBJ = OpenIPMI_wrap.o OpenIPMI_perl.o
+OpenIPMI_OBJ = OpenIPMI_wrap.lo OpenIPMI_perl.lo
 
 OpenIPMI.so: $(OpenIPMI_OBJ)
-	$(LIBTOOL) --mode=link $(CC) -shared -o $@ $^ $(OPENIPMI_SWIG_LIBS)
-	rm OpenIPMI.so
+	$(LIBTOOL) --mode=link $(CC) -shared -module -avoid-version -o $(@:.so=.la) $^ $(OPENIPMI_SWIG_LIBS) -rpath $(PERL_INSTALL_DIR)/auto/OpenIPMI
+	rm -f OpenIPMI.so
 	mv .libs/OpenIPMI.so .
 	rm -rf .libs
 
-OpenIPMI_wrap.o: OpenIPMI_wrap.c OpenIPMI.h
-	$(COMPILE) -c $<
+OpenIPMI_wrap.lo: OpenIPMI_wrap.c OpenIPMI.h
+	$(LIBTOOL) --mode=compile $(COMPILE) -c $<
 
-OpenIPMI_perl.o: OpenIPMI_perl.c OpenIPMI.h
-	$(COMPILE) -c $<
+OpenIPMI_perl.lo: OpenIPMI_perl.c OpenIPMI.h
+	$(LIBTOOL) --mode=compile $(COMPILE) -c $<
 
 OpenIPMI_wrap.c OpenIPMI.pm: $(top_srcdir)/swig/OpenIPMI.i OpenIPMI_lang.i
 	$(SWIG) $(DEFS) -perl5 -o $@ -I$(top_srcdir)/swig/perl $<
